src/main/java/com/google/step/snippet/external/W3SchoolsClient.java

```diff
@@ -6,12 +6,32 @@
 import org.jsoup.nodes.Document;
 import org.jsoup.select.Elements;
 
-public final class W3SchoolsClient {
+public final class W3SchoolsClient implements Client {
   private static final String TITLE_TAG = "h1";
   private static final String DESC_TAG = "p";
   private static final String SNIPPET_CLASS = "w3-example";
   private static final String CODE_CLASS = "w3-code";
 
+  private String cseId = null;
+
+  public W3SchoolsClient() {}
```

Comment:
Please remove unused empty constructor

Reply:
@kaylaleung could you address this? when i remove this part, it breaks your tests

Reply:
I think you can fix the test by just passing "UNUSED_CSEID_FOR_TEST" or something like that in the constructor